components/well/blender/recipe.js

```diff
@@ -0,0 +1,35 @@
+import { GEL } from '@westpac/core';
+import React from 'react';
+
+import { Well } from '@westpac/well';
+
+export function AllStyles({ brand }) {
+	return (
+		<GEL brand={brand}>
+			<Well>Text</Well>
```

Comment:
Good call not to run nested Wells in here as they use css selectors but this may break in the future...
I would make sure what you document below is also in here at the least.